Explorer/Assets/DCL/Multiplayer/Movement/Settings/MessageEncodingSettings.cs

```diff
@@ -4,7 +4,6 @@
 
 namespace DCL.Multiplayer.Movement.Settings
 {
-    [CreateAssetMenu(fileName = "MessageEncodingSettings", menuName = "DCL/MessageEncodingSettings")]
```

Comment:
Is there another way to instantiate this file?

Reply:
Once settings asset is created there is no need to have this menu, it only adds noise. So following to @Kinerius proposed approach, we always remove `CreateAssetMenu` after singleton asset being created. It basically keeps the menu clean.